activation/handler_v2_test.go

```diff
@@ -1813,64 +1810,6 @@ func Test_MarryingMalicious(t *testing.T) {
 	}
 }
 
-func TestContextualValidation_DoublePost(t *testing.T) {
```

Comment:
How about keeping the test to make sure this kind of malfeasance is detected?

Reply:
I will add it as a sub-test to `TestContextual_PreviousATX`. I want to extend this test to work with both an ATXv2 and an ATXv1 collision and then it would be 3 related tests grouped together.

Reply:
Added it as a new sub-test to `TestContextual_PreviousATX`